x-pack/plugins/data_visualizer/public/application/index_data_visualizer/hooks/use_data_visualizer_grid_data.ts

```diff
@@ -200,7 +200,16 @@ export const useDataVisualizerGridData = (
       const nonAggregatableFields: string[] = [];
 
       const fields = currentDataView.fields;
+
       fields?.forEach((field) => {
+        // Skip unsupported new field bytes_counter (#148518) for kibana_sample_data_logs data view
+        // which is technically a long/numeric field but is not supported for certain aggs
+        if (
+          currentDataView.getIndexPattern() === 'kibana_sample_data_logs' &&
```

Comment:
This check is specific to the Kibana Sample Data Logs data set. Is it possible to make this check more generic, for example by checking if there is a `time_series_metric` property in the field mapping equal to `counter`?

```
        "bytes_counter": {
          "type": "long",
          "time_series_metric": "counter"
        },
```

Could we treat these types of fields like non-aggregatable text fields where we show some example field values?

Reply:
Given that `"time_series_metric": "counter"` is a wider problem for Kibana, would it be best to revert the changes in this file in this PR, and merge the PR with the other 3 files? That would still be a useful fix right?

And then the `"time_series_metric": "counter"` problem can be fixed in a more holistic Kibana-wide way in a different PR.